velox/exec/SortedAggregations.h

```diff
@@ -31,7 +31,12 @@ class SortedAggregations {
   /// @param inputType Input row type for the aggregation operator.
   /// @param pool Memory pool.
   SortedAggregations(
-      std::vector<AggregateInfo*> aggregates,
+      const std::vector<const AggregateInfo*>& aggregates,
+      const RowTypePtr& inputType,
+      memory::MemoryPool* pool);
+
+  static std::unique_ptr<SortedAggregations> create(
```

Comment:
Please, add comments to explain that this method may return null.

Reply:
Added.